pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -28,12 +28,15 @@ import (
 	kubevirtv1 "kubevirt.io/api/core/v1"
 	cdiv1beta1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
 
+	"net/url"
```

Comment:
Please group stdlib imports together. [gimps](https://github.com/xrstf/gimps) can help you with that.

Reply:
I have gimpsed it. Thanks.